src/i18n/translations.ts

```diff
@@ -466,6 +472,19 @@ export default {
         undo: 'Undo',
         redo: 'Redo'
       },
+      Attribution: {
+        title: 'Properties',
+        add: 'Add Properties',
+        select: 'Please select a geometry first.',
+        submit: 'Submit'
+      },
+      AttributionRow: {
+        missingKey: 'Missing Key',
+        missingValue: 'Missing Value',
+        keyPlaceholder: 'Key',
+        valuePlaceholder: 'Value',
+        keyInUse: 'Key already exists!'
+      },
```

Comment:
This block seems not be translated in `de` yet.

Reply:
Good hint! a translations is now added